docs/whats-new/production-release-notes.md

```diff
@@ -1,5 +1,55 @@
 # Production release notes
 
+## Production release 31 August 2022
+Frontend version:  | Backend version:  
+**Updates** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Email reminders for expiring/expired secrets and certificates</summary>
+
+A new cron job sends email reminders to all Tenant admins whenever an application's certificate or secret is going to expire or if expired already. This email will prompt tenant admins to upload new certificate or create a new secret for the application.
+
+You will have up to 30 days to upload a new certificate or generate a new secret upon receiving such emails. Do this on time for your published applications; otherwise users of this application will have issues in accessing it.
+
+**Action required**: If you receive email notifications for expired or expiring certificates or secrets, please follow the instructions in the TechPass tenant guide to [upload new certificate](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/concepts/clientcred?id=upload-certificate) or [create new secret](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/concepts/authcodegrant?id=_2-create-secret).
+
+</details>
+
+**Fixes** - **TechPass Portal**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Implicit grant settings were overwritten when updating other application settings</summary>
+
+A fix has been applied so that if a Tenant Admin enables the ID Token in the **Implicit Grant settings** via Azure Portal and proceeds to change any application settings in the TechPass portal, the changes in the Azure Portal will be discarded.
+
+</details>
+
+**Fixes** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Email notification sent to deleted account indicate five days of no sign-in but it should be 30 days</summary>
+
+A fix has been applied to the email template to indicate 30 days instead of 5 days of no sign-in. It was only a typo, the logic for the deletion is triggered after 30 days, as intended.
+
+</details>
+
+**Fixes** - **Automation API**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Invite and Get user APIs does not return any value for UserPrincipalName</summary>
+
+On a rare occasion, Azure may take up more time than expected to generate a user resource when invite user API is triggered. On such occasions, Invite and Get user APIs may return no value for **UserPrincipalName** .
+
+A fix has been applied to manage the delay from Azure and to return a valid error message when UserPrincipalName is empty for the following APIs.  
+
+**Invite user API**
+[Invite Public Officer](https://stg.docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/IAM/paths/~1iam~1namespace~1{namespace}~1users~1publicofficer/post)  
```

Comment:
these four URLs should point to the production DocPortal (i.e. https://docs.developer.tech.gov.sg)

Reply:
fixed, thanks!